assets/promptflow/evaluators/models/coherence-evaluator/spec.yaml

```diff
@@ -5,8 +5,8 @@ properties:
   is-promptflow: true
   is-evaluator: true
   show-artifact: true
-  _default-display-file: ./evaluator/flow/prompt.jinja2
+  _default-display-file: ./evaluator/_coherence.py
 tags:
   Preview: ""
   hiddenlayerscanned: ""
-version: 1
\ No newline at end of file
+version: 2
```

Comment:
I think we should check all the descriptions.md, and do the qa->qr rename as needed.

Reply:
Yes - all the descriptions need to be reviewed in another commit. I need Ankit's help on this since it was his work item, but he was busy Friday so I'll check with him today.

Reply:
Minsoo signed off and the qa -> qr changes have been made.